gtk/src/default/gtk-3.20/_apps.scss

```diff
@@ -64,7 +64,7 @@ $ambiance: null !default;
     }
 
     notebook > header {
-      background: if($variant==light, $porcelain, $bg_color);
+      background: if($variant=='light', $porcelain, $bg_color);
```

Comment:
I think bg_color is already porcelain but I did not put the variable in the reorganize commit *selfslap*
So you could prly just write $bg_color and adjust the $bg_color to use $porcelain just to be a bit cleaner